.travis.yml

```diff
@@ -13,18 +9,51 @@ cache:
 env:
   global:
     - JACKRABBIT_VERSION=2.12.0
+    - SYMFONY__PHPCR__TRANSPORT=doctrinedbal
+    - TEST_FLAGS=""
+    - SYMFONY__DATABASE__DRIVER=pdo_mysql
+
+matrix:
+  include:
+    - php: 5.5
+    - php: 7.0
+      env: 
+        - SYMFONY__PHPCR__TRANSPORT=jackrabbit
+        # restart jackrabbit after each suite see: https://github.com/sulu-io/sulu/issues/2137
+        - TEST_FLAGS="--jackrabbit-restart"
+    - php: 7.0
+      env:
+        - SYMFONY__DATABASE__DRIVER=pdo_sqlite
+        - SYMFONY__DATABASE__PATH=test.sqlite
+    - php: 7.0
+      env: 
+        - SYMFONY__DATABASE__DRIVER=pdo_pgsql
+        - SYMFONY__DATABASE__USER=postgres
+        - SYMFONY__DATABASE__PASSWORD=postgres
+  allow_failures:
```

Comment:
I would not add these... Nobody looks at them anyway, they won't work if we do something about it, and it delays the build process a lot.